elasticdl/pkg/commonnew/embedding_tabel.go

```diff
@@ -0,0 +1,70 @@
+package commonnew
+
+import (
+	"elasticdl.org/elasticdl/pkg/proto"
+	"github.com/tensorflow/tensorflow/tensorflow/go/core/framework/types_go_proto"
+)
+
+// EmbeddingTable struct
+type EmbeddingTable struct {
+	Dim              int64
+	Initializer      string
+	EmbeddingVectors map[int64]*Tensor
+	Dtype            types_go_proto.DataType
+}
+
+// NewIndexedSlices return proto.IndexedSlices
+func NewIndexedSlices(t *Tensor, indices []int64) *proto.IndexedSlices {
```

Comment:
Where is this function used?
It seems that it's a very thin wrapper, we could define a `proto.IndexedSlices` directly.

Reply:
This function is moved to Tensor. This is actually the Serializer function.